### DISCUSSION TITLE
jobrunr-spring-boot-3-starter in version 6.1.x

### DISCUSSION THREAD

**Discussion by francoisxavierverryser** [jobrunr/jobrunr#727] [General]
Hello
We would like using job filters to make slack notifications on job events.
Using jobrunr with spring boot, version 6.1.0 is not yet available with jobrunr-spring-boot-3-starter dependency.
Do you plan to release a 6.1.x version of jobrunr-spring-boot-3-starter including these latest features ?
Thanks.

**Comment by jhchee** [comment]
6.1.1 is already on maven.
```
<dependency>
  <groupId>org.jobrunr</groupId>
  <artifactId>jobrunr-spring-boot-3-starter</artifactId>
  <version>6.1.1</version>
</dependency>
```

**Reply by francoisxavierverryser** [reply]
Indeed it is now.
Thanks a lot for the reply
